src/ui/Facet/FacetSearchElement.ts

```diff
@@ -128,11 +126,10 @@ export class FacetSearchElement {
     });
   }
 
-  public positionSearchResults(root: HTMLElement, facetWidth: number, nextTo: HTMLElement) {
+  public positionSearchResults() {
```

Comment:
These three parameters weren't necessary anymore since `PopupUtils` isn't used here anymore. If you want to know what impacts the removal of each of those had on the rest of the code base, they were removed individually across 3 separate functional commits.